lib/onEvent/message.js

```diff
@@ -2,35 +2,81 @@ const Kirbi = require('../../../../kirbi');
 
 //initialize AI
 if (Kirbi.Config.elizaEnabled && !Kirbi.Eliza) {
-	var Eliza = Kirbi.require('./extras/eliza');
+	let Eliza = Kirbi.require('./extras/eliza');
 	Kirbi.Eliza = new Eliza();
 	console.log('Eliza enabled.');
 	Kirbi.Eliza.memSize = 500;
 };
 
+function currentChannelHasHook (channelId) {
+	if (Kirbi.Auth.slack.webhooks[Kirbi.Slack.dataStore.getChannelById(channelId).name]) return true;
+	return false;
+};
+
 function slackMessageCb (output, msg, expires, delCalling) {
 	//we need to format the message output for the Slack API first...
-	let reformatted = {};
+	if (typeof output === 'object') {
+		let reformatted;
+		reformatted = {};
+		reformatted.attachments = [];
+		reformatted.attachments[0] = {};
+		reformatted.attachments[0].fallback = 'There was an error';
```

Comment:
also since the actual assignment isn't being changed, this should be a const instead of a let